GitUI/GitUI.csproj

```diff
@@ -86,6 +90,10 @@
     <Content Include="Themes\dark.css">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Can we instead replace all the `<Content>`s by only one `<Content Include="Themes\*.css">`?

/cc @RussKie @NikolayXHD

Reply:
Yes, we can